flows/definition/migrations/13_1.go

```diff
@@ -0,0 +1,26 @@
+package migrations
+
+import (
+	"github.com/nyaruka/goflow/utils/uuids"
+
+	"github.com/Masterminds/semver"
+)
+
+func init() {
+	registerMigration(semver.MustParse("13.1"), AsParsed(Migrate13_1))
+}
+
+// Migrate13_1 adds UUID to send_msg templating
+func Migrate13_1(f Flow) Flow {
+	for _, node := range f.Nodes() {
```

Comment:
We can totally do this later but it seems this type of pattern (find me all the types of this node, let me tweak it and return it) is going to be super popular so having a utility method in time will be wanted.

IE, I can imagine something where you just give it the type of action you want and a receiver method. That method returns the modified thing (or nil) and the utility function takes care of fixing things up appropriately in the case of removal etc..

But this is a good baby step.